src/main/java/ticketservice/dao/TicketDao.java

```diff
@@ -1,26 +1,30 @@
-package dao;
+package ticketservice.dao;
 
-import jdbc.CustomDataSource;
-import ticket.Ticket;
-import ticket.TicketType;
+import org.springframework.stereotype.Component;
+import ticketservice.ticket.Ticket;
+import ticketservice.ticket.TicketType;
 
+import javax.sql.DataSource;
 import java.sql.*;
 import java.util.ArrayList;
 import java.util.List;
 
+@Component
```

Comment:
for DAO level it is better to use specific annotation: `@Repository`

Reply:
Fixed